Site/Repository/LayoutRepository.php

```
@@ -212,7 +212,7 @@ public function removeThumbnail(Layout $layout, BBApplication $app)
         $thumbnailfile = $layout->getPicPath();
         File::resolveFilepath($thumbnailfile, null, array('include_path' => $app->getResourceDir()));
 
-        while (true === file_exists($thumbnailfile) && true === is_writable($thumbnailfile)) {
+        while (true === is_file($thumbnailfile) && true === is_writable($thumbnailfile)) {
             @unlink($thumbnailfile);
```

No because picpath is not required value, then we can't throw an exception because this value is not set. Them the side effect of add an acception : I can't delete a Layout because I don't set an non required value, this is not friendly.